### ISSUE TITLE
Support key ranges

### ISSUE THREAD

**Issue by vladimir-e** [null-null-null/ember-keyboard#13]
After learning the code I don't think there is a simple way to listen to a key range like `A-z0-9\ `. Let me describe a use case for that.

I have a `table-cell` component (`<td>`) which can be "selected" by clicking on it. And now when it's selected I want user to be able to start typing so text inside the cell should be replaced with input field. 

``` JavaScript
  // app/components/editable-table-cell/component.js

  editOnType: Ember.on(keyUp('Alphanumeric'), function() {
    this.startEditing();
  }),

  cancelOnEsc: Ember.on(keyUp('Escape'), function() {
    this.cancelEditing();
  }),
```

``` JavaScript
  // Or even like that
  editOnEnter: Ember.on(keyUp('Alphanumeric', 'Enter'), function() {
    this.startEditing();
  }),
```

Do you think it would be useful to have? I could work on that.

**Comment by null-null-null** [comment]
That's an interesting use case, though I can't think of a way to support it without adding significant complexity. Let me think about this some more, though. There might be a solution I'm not considering yet.

**Comment by vladimir-e** [comment]
Hi there,

I think It's a very elegant solution. It doesn't add complexity for a regular (one-key) workflow at all, but adds a lot of flexibility introducing `keyUp()` with no arguments. Cool. Do you want me to give it a try?

**Comment by null-null-null** [comment]
I've got a good idea of how to do it, but if you want to take a crack at it, I'd be happy to just review/offer guidance. Let me know what you decide.

**Comment by vladimir-e** [comment]
Oh, I can see you actively working on the addon these days. Don't want to hold you on this task, as I will be able to get to it next week only.

**Comment by null-null-null** [comment]
Resolved by https://github.com/null-null-null/ember-keyboard/commit/d3fa15aa5c5ed91ff2987f49ab74f30289e87edb. Let me know if it works for you!

**Comment by vladimir-e** [comment]
Just checked it out and it works great. Thank you!

Couple of fixes for README piece of this commit:
1. `getKey` expects `event`, not `keyCode`, so in current example you would get `undefined` all the time.
2. Closing bracket is missing on `if` that checks regex.
(not sure if that worth opening PR)

I ended up writing this code for the original case:

``` JavaScript
import { keyDown, getKey } from 'ember-keyboard';

// ...

editOnType: Ember.on(keyDown(), function(event) {
  let keyName = getKey(event);
  if (/^[a-z0-9\ \-\=\/\.\,\']{1}$/.test(keyName)) {
    this.set("value", keyName);
    this.startEditing();
  }
}),
```

Concerns somewhat related to this ticket:
- there's no easy way to detect key combinations, for example "cmd+f" would fire observer two times, so second event is indistinguishable from pressing `f` key alone. That way OS/browser combinations interfere with component's hotkeys.
- also it is difficult to track capital letters and some special characters, because `Shift` key fires separately.

But I'll get back to that and hopefully submit a PR.

Great job on this add-on!
